lib/debug/session.rb

```diff
@@ -953,12 +958,20 @@ def self.require_location
     nil
   end
 
+  def self.prepend_console_for_test
+    return unless ENV['RUBY_DEBUG_TEST_MODE']
+
+    require_relative 'test_console'
+    UI_Console.prepend(TestUI_Console)
+  end
+
   # start methods
 
   def self.console **kw
     set_config(kw)
 
     require_relative 'console'
+    prepend_console_for_test
```

Comment:
`prepend` is implementation (HOW TO).
It should be `prepare` or something goal (WHAT DOES IT).

BTW, only 4 lines, this method can be inlined here.

Reply:
I will be inline them.

Reply:
https://github.com/ruby/debug/pull/48/files#diff-c87adef8ace662a03249d206e6812162256bbcecabe18cd51861d1e1cd9cdfcaR965-R968